sdks/java/harness/src/main/java/org/apache/beam/fn/harness/FnApiDoFnRunner.java

```diff
@@ -235,6 +251,12 @@
    */
   private StateKey.BagUserState cachedPartialBagUserStateKey;
 
+  /**
+   * This member should only be accessed indirectly by calling
+   * {@link #createOrUseCachedMultimapSideInputKey} and is only valid during {@link #processElement}
```

Comment:
<!--new_thread; commit:d1c0ffde59d0b17126deecb840cbeec36719d7a6; resolved:0-->
The pattern of naming `createOrUseX()` should just be `getX()`, here and throughout the file and our codebase. Not only is it less verbose, it properly encapsulates the abstraction of whether or not it is lazily built.

Reply:
Done.